### ISSUE TITLE
travis release android builds(maybe others) are debug builds.

### ISSUE THREAD

**Issue by zethfoxster** [WagicProject/wagic#670]
anyway to get the auto builds from travis to do release builds for android?
havent checked others i would assume theyre being made as debug builds too.

**Comment by xawotihs** [comment]
Sounds like Travis/Appveyor releases on Mac and Windows contains the testsuite as well and should not.

**Comment by zethfoxster** [comment]
Yeah the preformance difference from a true release build compared to debug is huge, very noticable.

**Comment by xawotihs** [comment]
Windows Appveyor build does not show the test suite and is actually built in release. One checked.

**Comment by zethfox** [comment]
Noticed i cant get a true release build manually either. I go through the process, enter the password, install the release version apk...test suite and ai testing in menu.

**Comment by zethfoxster** [comment]
Travis_scripts.sh, the scripts here should be for "release" not "debug"

**Comment by zethfoxster** [comment]
if [ "$BUILD_ANDROID" = "YES" ]; then
    android-ndk-r9/ndk-build -C projects/mtg/Android -j4
    $ANDROID list targets
    $ANDROID update project -t 1 -p projects/mtg/Android
    ant debug -f projects/mtg/Android/build.xml
fi

Notice its building for debug

**Comment by zethfoxster** [comment]
https://gist.github.com/mariotaku/7a0c51955d14def2fa0e

Not sure if this would help us

**Comment by zethfoxster** [comment]
hey Xaw, is there anyway you can help us with this one?
Ive tested 2 devices with both a personally built release build and a "release" debug build off our auto builds, the difference in preformence is astronomical.

**Comment by xawotihs** [comment]
If you know what you changed to create your personal release built, we should be able to put it in Travis.

**Comment by zethfoxster** [comment]
I used command prompt, and where i mentioned earlier in this report i wrote "release" instead of debug.

But how do you tell travis to use a keystore?

**Comment by zethfoxster** [comment]
you are my heros, thank you. Android is not debug anymore from what im seeing.
it wasnt so much about hiding the test suite options and stuff, but the difference between the release vs debug in preformence.
now we just need to see if the rest of the releases are also not debug.

**Comment by zethfoxster** [comment]
Android, psp, win are good.
Can an iphone/linux user confirm this is fixed.
